### DISCUSSION TITLE
Github Copilot stopped having access to my workspace editor in VSCode?

### DISCUSSION THREAD

**Discussion by Fontanapink** [community/community#102854] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Starting today, GitHub Copilot chat responds with the following:

"I'm sorry, but I can't check your script for errors without seeing the actual code. Could you please share the code that you want me to check?"

or

"I'm sorry for the confusion, but as an AI, I don't have access to your local files or active selection. Could you please paste the relevant parts of your script here?"

Even when I am selecting the code in VS Code Editor, or if I start my prompts with /workspace

Github Copilot version v1.156.0
VS Code version 1.85.2

**Comment by Fontanapink** [comment]
and GitHub Copilot Chat v0.11.1

**Comment by f-corvaro** [comment]


**Comment by cfognom** [comment]
Same problem for me started happening after I updated GitHub Copilot Chat from v0.11.1 to v0.12.0. The solution was to revert the update back to v0.11.1. After that I could just open a new chat and it could reference my code again.

If it is v0.11.1 in your case that is causing the issues, perhaps try downgrading your version to v0.10.2 and see if it helps.

**Reply by cfognom** [reply]
Hmm, it seems this only made it work for one message, afterwhich it started spewing out "Cannot read properties of undefined (reading 'split')". Back to the drawing board.

**Comment by vedavrat-shortcastle** [comment]
**Facing similar issue**

**Environment:**

* **IDE:** Visual Studio Code
* **VS Code Version:** 1.86.0
* **GitHub Copilot Extension Version:** v1.172.0
* **GitHub Copilot Chat Extension Version:** v0.13.0

**Problem Description:**

I've encountered issues with the GitHub Copilot Chat extension. Specifically, the following features are not working as expected:

* **Copilot Chat cannot read project files:** This prevents core functionality where Copilot Chat should be able to understand my code's context.
* **@workspace feature is unresponsive:** Commands reliant on understanding the workspace context are likely failing.
* **/explain feature is not working:** I cannot ask Copilot Chat to explain code snippets or portions of my project.

**Expected Behavior:**

GitHub Copilot Chat should be able to analyze my project files and provide context-aware assistance, including workspace-related features and code explanations.

**Comment by kasperk81** [comment]
it's funny that plugin is installed in the IDE and chat window shows that it's sharing the files with copilot, chatgpt, and other six AIs, but NONE of them seem to agree that they received the file. try debating and they go "I understand your frustration.."

**Comment by desinox** [comment]
Its working in the latest pre-release (v0.16.2024052001)

**Comment by Dats4794** [comment]
I'm getting this bug. The latest github copilot can't access my workspace. Does anyone have a fix for this?

**Comment by PatrickLKJ** [comment]
try #codebase , it works for me

**Comment by martinbogo** [comment]
Make sure you haven't flipped accidentally from "Agent" mode to "Ask" mode

**Comment by inghak** [comment]
In "classic" Visual Studio, try #solution in place of @workspace
